deepchecks/vision/simple_classification_data.py

```diff
@@ -22,38 +22,47 @@
 
 from deepchecks import vision
 
-__all__ = ['load_dataset', 'SimpleClassificationDataset', 'SimpleClassificationData']
-
-
-def load_dataset(
-    root: str,
-    train: bool = True,
-    batch_size: int = 32,
-    num_workers: int = 0,
-    shuffle: bool = True,
-    pin_memory: bool = True,
-    object_type: Literal['VisionData', 'DataLoader'] = 'DataLoader',
-    **kwargs
-) -> t.Union[DataLoader, vision.ClassificationData]:
+__all__ = ['classification_dataset_from_directory', 'SimpleClassificationDataset', 'SimpleClassificationData']
+
+
+def classification_dataset_from_directory(
+        root: str,
+        batch_size: int = 32,
+        num_workers: int = 0,
+        shuffle: bool = True,
+        pin_memory: bool = True,
+        object_type: Literal['VisionData', 'DataLoader'] = 'DataLoader',
+        **kwargs
+) -> t.Union[t.Tuple[t.Union[DataLoader, vision.ClassificationData]], t.Union[DataLoader, vision.ClassificationData]]:
     """Load a simple classification dataset.
 
     The function expects that the data within the root folder
-    to be structured in the following way:
+    to be structured one of the following ways:
 
         - root/
             - train/
                 - class1/
                     image1.jpeg
+            - validation/
+                - class1/
+                    image1.jpeg
             - test/
                 - class1/
                     image1.jpeg
 
+        - root/
+            - train/
+                - class1/
+                    image1.jpeg
+
+        - root/
+            - class1/
+                image1.jpeg
+
```

Comment:
I think this part is not clear enough. 
Also I'd start with the shorter version first.